### DISCUSSION TITLE
ipv6 support?

### DISCUSSION THREAD

**Discussion by 645340633** [hassio-addons/addon-node-red#1192] [Ideas]
Hi.
I have installed node-red at hassos
My internet-provider use ds-lite which only use IPv6 to access from outside.
How can i access node-red with ipv6.
Normally ipv4 "199.0.0.1:1880" works
Example IPv6-adress is 1a0:2a02:9018:2221:5120:96d8:fb11:5815.
ipv6 "[21a0:2a02:9018:2221:5120:96d8:fb11:5815]:1880" does not work for access to node-red.
ipv6 "[21a0:2a02:9018:2221:5120:96d8:fb11:5815]:80" does work for access to web server.
It seems, that node-red does not listen to IPv6, only IPv4.
What and where do i have to adjust for solving the problem?
Thank's
greetings Uw

**Comment by 8191** [comment]
Any update on this? It seems that IPv6 is still not possible. Considering that many ISP do not support externally exposed IPv4 addresses anymore (CGNAT) this feature might help many users...

**Comment by 8191** [comment]
As a temporary workaround until the add-on get's modified, you can add following init_commands:

```
sed -i s/127.0.0.1/::/ /etc/node-red/config.js
sed -i 's/listen {{ .port }} \(.*\);/listen [::]:{{ .port }} \1;/' /etc/nginx/templates/direct.gtpl
```
